src/server/graphql/models/Team/endMeeting/endMeeting.js

```diff
@@ -136,7 +138,20 @@ export default {
             meetingPhaseItem: null,
           });
       });
-
+    // dispatch segment events:
+    const segmentTraits = await getSegmentTraitsForUsers(
```

Comment:
should we use standard `then` syntax because none of these are on the critical path?